kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java

```diff
@@ -1035,12 +1055,19 @@ private void collectAttributes(Map<String, Boolean> attributes, Att att) {
     }
 
     private static final Production INJ_PROD = Production(KLabel(KLabels.INJ, Sort("From"), Sort("To")), Sort("To"), Seq(NonTerminal(Sort("From"))));
+    private static final Production CEIL_PROD = Production(KLabel(KLabels.ML_CEIL.name(), Sort("From"), Sort("To")), Sort("To"), Seq(NonTerminal(Sort("From"))));
```

Comment:
These should not be necessary as we now have productions in the grammar for them. That's what computePolyProd is for.